### DISCUSSION TITLE
Compliments on the build process

### DISCUSSION THREAD

**Discussion by JudgeGroovyman** [aardappel/lobster#332] [General]
I just wanted to express my appreciation for streamlining the build process.  I have been testing other engines and frameworks and have been finding enormous issues just getting the simplest stuff working, but not with Lobster.  It worked perfectly even on my M1 Macbook.  The whole process was easy and went flawlessly exactly as described.  

So anyway ... thank you all!

**Comment by JudgeGroovyman** [comment]
Oh!  I'm just putting together a connection: I programmed a simple app for the Amiga just last year in Amiga E and apparently Amiga E was originally written by Mr Oortmerssen as well as this language Lobster.  

Also my cousin Chris Natsuume worked on the original FarCry and I believe he worked with Mr Oortmerssen on that as well.

Sometimes its a small world :)

**Reply by aardappel** [reply]
Wow, that is a small world! I don't think there'd be a lot of people trying out E and Lobster independently (without knowing they're both made by me).

And yes I worked with Chris :)
